README.md

```diff
@@ -155,7 +155,7 @@ NOTE: The steps need to be done in order to create a new Hyrax based app.
 Generate a new Rails application using the template.
 
 ```
-rails _5.2.4.3_ new my_app -m https://raw.githubusercontent.com/samvera/hyrax/v3.0.0-rc2/template.rb
+rails _5.2.4.3_ new my_app -m https://raw.githubusercontent.com/samvera/hyrax/v3.0.0.pre.rc2/template.rb
```

Comment:
i think our preferred tag format, per [SemVer](https://semver.org/#spec-item-9) is `-rc2`. maybe the best fix here is to ensure we can push a tag like that?

Reply:
this link now works: https://raw.githubusercontent.com/samvera/hyrax/v3.0.0-rc2/template.rb